lib/io/file/stat.fz

```diff
@@ -67,7 +67,7 @@ private stat(
       meta_data data[0] data[1] (data[2] == 1) (data[3] == 1)
     else
       replace
-      error "error while retrieving meta_data for the following file/dir: $path"
+      error "error number: {data[0]}"
```

Comment:
Don't you think it helps to include `$path` as well?

Reply:
sure, the caller will know this. But the caller will perform lazy error handling and propagate an error further up to the caller's caller who does not know :-)